requirements.txt

```diff
@@ -2,32 +2,32 @@
 
 # If you update these, also update iati_datastore/setup.py
 # These are duplicated here so that requires.io will flag out of date dependencies
-Flask==0.10
-Jinja2==2.6
-SQLAlchemy==0.8
-Flask-SQLAlchemy==1.0
-Werkzeug==0.8.3
-ckanapi==1.5
-iso8601==0.1.4
-lxml==3.4.1
-psycopg2==2.4.6
-python-dateutil==2.1
-six==1.10.0
-unicodecsv==0.9.4
-voluptuous==0.7.1
-Flask-Script==0.5.3
-prettytable==0.7
-alembic==0.5.0
-gunicorn==0.17.2
-defusedxml==0.4
-redis==2.7.2
-rq==0.3.7
-Unidecode==0.04.12
-requests==2.5.3
-Flask-RQ==0.2
-flask-heroku==0.1.4
-Flask-And-Redis==0.4
-Flask-Markdown==0.3
-xmltodict==0.7.0
-gevent>=0.13.8
+Flask==0.10  # rq.filter: <0.11
+Jinja2==2.6  # rq.filter: <3.0
+SQLAlchemy==0.8  # rq.filter: <0.9
+Flask-SQLAlchemy==1.0  # rq.filter: <2.0
+Werkzeug==0.8.3  # rq.filter: <0.9
+ckanapi==1.5  # rq.filter: <2.0
+iso8601==0.1.4  # rq.filter: <0.2
+lxml==3.4.1  # rq.filter: <4.0
+psycopg2==2.4.6  # rq.filter: <3.0
+python-dateutil==2.1  # rq.filter: <3.0
```

Comment:
Looks like this should say:

```
python-dateutil==2.1  # rq.filter: <2.5
```

Reply:
Assuming SemVer (which [they follow](https://github.com/dateutil/dateutil/blob/1119b48e805bae531a0e64b061972f59bfb4df16/RELEASING#L39)), should be `3.0` since that's the next integer and anything until then should be backwards compatible.

Reply:
> (which they [follow](https://github.com/dateutil/dateutil/blob/1119b48e805bae531a0e64b061972f59bfb4df16/RELEASING#L39))

Well, they say “try to”! And I’m not saying they don’t… I suspect this problem was due to an undocumented feature that was later removed (I don’t know if that breaks semver, but I guess not).

So a good way to test this PR is:

 * assume requirements.io will update to the latest version that matches the filter
 * strip out `==[^ ]+  # rq.filter: ` from every line of the requirements files
 * create a fresh virtualenv, and install with pip
 * run tests

If you do that, you’ll find some tests fail, due to this:

```
from dateutil.parser import parse
parse("")  # this succeeds on versions earlier than 2.5
```

There’s also one more failing test, that looks a bit more serious (`offset`s being ignored)… I’m just investigating that one!

Reply:
Based on @andylolz 's write-up, I'd also suggest `python-dateutil==2.1  # rq.filter: <2.5`